test/e2e/security_test.go

```diff
@@ -148,7 +149,7 @@ func testCert(t *testing.T, data *TestData, expectedCABundle string, restartPod
 	}
 
 	var caBundle string
-	if err := wait.Poll(2*time.Second, timeout, func() (bool, error) {
+	if err := wait.Poll(20*time.Second, timeout, func() (bool, error) {
```

Comment:
timeout is 10s, while the interval is 20s in restartPod case, then there should be only 1 try. I don't think it is as your expected.

Reply:
Thanks for reminding me. Yes, I only paid attention to the 2m case. It is better to keep it as it was.